modules/installation-configuration-parameters.adoc

```diff
@@ -1074,251 +1074,512 @@ within link:https://azure.microsoft.com/en-us/global-infrastructure/regions[a re
 ====
 
 .Additional Azure parameters
-[cols=".^2,.^3a,.^3a",options="header"]
+[cols=".^2l,.^3a,.^3a",options="header"]
 |====
 |Parameter|Description|Values
 
-|`compute.platform.azure.encryptionAtHost`
+|compute:
+  platform:
+    azure:
+      encryptionAtHost:
```

Comment:
Probably doesn't matter, but all of `platform` parameters are indented a single space; all the rest are two spaces.

Reply:
the 2nd line is actually indented two spaces, it just doesn't line up under the 3rd letter of the preceding line because the preceding line has a sneaky `|` character in it